pyproject.toml

```diff
@@ -0,0 +1,87 @@
+[tool.poetry]
+name = "foreshadow"
+version = "0.1.0"
+description = "Peer into the future of a data science project"
+authors = [
+    "Adithya Balaji <adithyabsk@gmail.com>",
+    "Alexander Allen <>"
+]
+license = "Apache-2.0"
+readme = "README.rst"
+homepage = "https://foreshadow.readthedocs.io"
+repository = "https://github.com/georgianpartners/foreshadow"
+documentation = "https://foreshadow.readthedocs.io"
+keywords = [
+    "data",
+    "machine",
+    "learning",
+    "automl",
+    "foreshadow"
+]
```

Comment:
add feature engineering maybe?

Reply:
We only get 6 keywords, would you change one of those?